examples/navigation/index.js

```diff
@@ -0,0 +1,30 @@
+import 'rxjs';
```

Comment:
@blesh it is done so that all the operators are placed on `Observable.prototype` because Epics receive an existing observable that they do not create. This is suggested in the docs: http://redux-observable.js.org/docs/basics/Epics.html#a-basic-example